package/kernel/linux/modules/netdevices.mk

```diff
@@ -110,6 +110,23 @@ endef
 $(eval $(call KernelPackage,mii))
 
 
+define KernelPackage/mdio-gpio
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:= Supports GPIO lib-based MDIO busses
+  DEPENDS:=+kmod-libphy @GPIO_SUPPORT
+  KCONFIG:=CONFIG_MDIO_BITBANG \
+		CONFIG_MDIO_GPIO
+  FILES:=$(LINUX_DIR)/drivers/net/phy/mdio-gpio.ko $(LINUX_DIR)/drivers/net/phy/mdio-bitbang.ko
```

Comment:
Please add a line break here

Reply:
@hauke 
Sorry was not only in the last 2 days. But you have already merged my PR thanks.
So i think this is OK?